Java/src/main/java/epam/Planes/PassengerPlane.java

```diff
@@ -0,0 +1,43 @@
+package epam.Planes;
+
+import java.util.Objects;
+
+
+public class PassengerPlane extends Plane {
+	private int passengersCapacity;
+
+	public PassengerPlane(String model, int maxSpeed, int maxFlightDistance, int maxLoadCapacity,
+			int passengersCapacity) {
+		super(model, maxSpeed, maxFlightDistance, maxLoadCapacity);
+		this.passengersCapacity = passengersCapacity;
+	}
+
+	public int getPassengersCapacity() {
+		return passengersCapacity;
+	}
+
+	@Override
+	public String toString() {
+		return super.toString().replace("}", ", passengersCapacity=" + passengersCapacity + '}');
+	}
+
+	@Override
+	public boolean equals(Object o) {
```

Comment:
change the name of variable "o", variable names must be self descriptive

Reply:
Done